typescript/ws-protocol/README.md

```diff
@@ -16,28 +14,13 @@ This package does not require a specific WebSocket server or client implementati
 $ npm install ws
 ```
 
-## Example servers
-
-### System monitor
-
-A system monitor script is provided as an illustrative example of a WebSocket server. To try it out:
-
-1. Run `npx @foxglove/ws-protocol-examples sysmon`.
-2. In a browser, open up https://studio.foxglove.dev and initiate a Foxglove WebSocket connection to `ws://localhost:8765/`.
-
-<img width="500" alt="Foxglove Studio displaying memory and CPU usage from the system monitor example" src="https://user-images.githubusercontent.com/14237/145313065-85c05645-6b29-4eb2-a498-849c83f8792d.png">
+## Example servers and client
 
-### Generic server template
+Run these [example servers and client](/typescript/ws-protocol-examples), implemented in JavaScript/TypeScript, to get started.
 
-The following script provides a simple example server which publishes messages on a single topic called `example_msg`, using JSON to encode message data and [JSON Schema](https://json-schema.org) to describe the message layout. This is meant to be a simple example from which you can begin to build a custom server.
+## Server template
 
-To get started with the example server:
-
-1. Copy the script below into a file named `server.js`.
-2. Run `node server.js`.
-3. In a browser, open up https://studio.foxglove.dev and initiate a Foxglove WebSocket connection to `ws://localhost:8765/`.
-
-<img width="676" alt="Foxglove Studio displaying data from the example server" src="https://user-images.githubusercontent.com/14237/145260376-ddda98c5-7ed0-4239-9ce4-10778ee8240b.png">
+The template below publishes messages on a single topic called `example_msg`, using JSON to encode message data and [JSON Schema](https://json-schema.org) to describe the message layout.
```

Comment:
I notice you kept this example code in the readme rather than linking out to a file. Is that because the example servers are more complex in this TS package than the Python one? Do you think we should put this simple example server in the TS ws-protocol-examples package as well and then just link to it? (That doesn't have to be decided/done in this PR, we can do it as follow-up work)

Reply:
I would prefer putting it in the `ws-protocol-examples` package and linking to it. Didn't do here, because I didn't see an existing file like for the Python package /  wasn't sure if there was a reason for that.

Reply:
I can do this soon or you're welcome to do it as well

Reply:
TODO: Keep template as is in README --> `ws-protocol-examples` repo.